src/platform/utilities/feature-toggles/featureFlagNames.json

```diff
@@ -256,5 +256,7 @@
   "virtualAgentEnableMsftPvaTesting": "virtual_agent_enable_msft_pva_testing",
   "virtualAgentEnableNluPvaTesting": "virtual_agent_enable_nlu_pva_testing",
   "yellowRibbonAutomatedDateOnSchoolSearch": "yellow_ribbon_automated_date_on_school_search",
-  "toggleVyeAdressDirectDepositForms": "toggle_vye_adress_direct_deposit_forms"
+  "toggleVyeAddressDirectDepositForms": "toggle_vye_address_direct_deposit_forms",
+  "toggleVyeAddressDirectDepositFormsInProfile": "toggle_vye_address_direct_deposit_forms_in_profile",
+  "vyeLoginWidget":" vye_login_widget"
```

Comment:
Looks to be a space at the beginning of the feature flag here that shouldn't be here.